test/e2e/handler/static_addr_and_route_test.go

```diff
@@ -26,7 +26,11 @@ import (
 	nmstate "github.com/nmstate/kubernetes-nmstate/api/shared"
 )
 
-func ipV4AddrAndRoute(firstSecondaryNic, ipAddress, destIPAddress, prefixLen, nextHopIPAddress string) nmstate.State {
+func ipV4AddrAndRoute(firstSecondaryNic, ipAddress, destIPAddress, prefixLen, nextHopIPAddress,
```

Comment:
Let's keep `ipV4AddrAndRoute` and add a new `ipV4AddrAndRouteẀithTableID`, the first one will call the second one with tableID "". Test without TableID are more common.

Reply:
Yep looks better, done